rubi/rubi/rubicon_types/orders.py

```diff
@@ -276,58 +313,6 @@ def __init__(self, pair_name: str, order_id: int):
         self.order_id = order_id
 
 
-class Transaction:
```

Comment:
to help keep `client` functions as useable, such as `place_market_order`, i believe we will want to keep this class until enough time has passed that we feel comfortable forcing this migration. 

with `Transaction` really just containing `orders: List[BaseOrder]` i think we should be able to make quick work of making the functions that we plan to deprecate composable with the new ones.